nautobot/dcim/api/serializers.py

```diff
@@ -326,7 +355,7 @@ class RackUnitSerializer(serializers.Serializer):
     occupied = serializers.BooleanField(read_only=True)
 
 
-class RackReservationSerializer(NautobotModelSerializer, TaggedModelSerializerMixin):
+class RackReservationSerializer(TaggedModelSerializerMixin, NautobotModelSerializer):
```

Comment:
Based on some very basic manual testing, changing the order here affects the order in which fields appear in the serialized JSON (specifically, whether `tags` comes before or after `notes_url` and `custom_fields`) but otherwise it appears either is presently valid.